### PR TITLE
Update IntelliJ version to 2019.1 RC

### PR DESCRIPTION
**Pull request by Flanker32**
This pr is used to build tooling bits with Intellij 2019.1 for test, will hold this pr until we decide to update to new version.

### CODE REVIEW

**Comment by Eskibear** [comment]
should include `BuildDevInt`, which is executed in PR build.

**Comment by Eskibear** [comment]
What's the consequence of skipping buildSearchableOptions?

**Comment by Flanker32** [comment]
@Eskibear Issues about buildSearchableOptions has been fixed with intellij 2019.1 rc and scala plugin:2019.1.2

File: BuildDevint [hunk 1]
```diff
@@ -17,11 +17,11 @@ BUILD_ECLIPSE=true
 BUILD_INTELLIJ=true
 INJECT_INTELLIJ_VERSION=false
 
-IJ_VERSION_LATEST=2018.3
-IJ_SCALA_VERSION_LATEST=2018.3.2
+IJ_VERSION_LATEST=191.6183.62
+IJ_SCALA_VERSION_LATEST=2019.1.2
```
Comment (Eskibear) [inline BuildDevint]: 2019.1.5   see https://plugins.jetbrains.com/plugin/1347-scala

File: BuildProduct.sh [hunk 2]
```diff
@@ -18,13 +18,13 @@ BUILD_INTELLIJ=true
 
 INJECT_INTELLIJ_VERSION=false
 
-IJ_VERSION_LATEST=2018.3
-IJ_DISPLAY_VERSION_LATEST=2018.3
-IJ_SCALA_VERSION_LATEST=2018.3.2
+IJ_VERSION_LATEST=191.6183.62
+IJ_DISPLAY_VERSION_LATEST=2019.1
+IJ_SCALA_VERSION_LATEST=2019.1.2
```
Comment (Eskibear) [inline BuildProduct.sh]: 2019.1.5  https://plugins.jetbrains.com/plugin/1347-scala

File: PluginsAndFeatures/azure-toolkit-for-intellij/gradle.properties [hunk 3]
```diff
@@ -2,6 +2,6 @@ javaVersion=1.8
 org.gradle.jvmargs='-Duser.language=en'
 sources=false
 intellij_version=IC-LATEST-EAP-SNAPSHOT
-dep_plugins=org.intellij.scala:2019.1.1
+dep_plugins=org.intellij.scala:2019.1.2
```
Comment (Eskibear) [inline PluginsAndFeatures/azure-toolkit-for-intellij/gradle.properties]: 2019.1.5

**Pull request merged**